packages/client/src/components/ProductListDropDown/DropDownViewList/DropDownViewList.component.js

```diff
@@ -1,39 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { apiURL } from '../../../apiURL';
 import DropDownView from '../DropDownView/DropDownView.component';
 import './DropDownViewList.style.css';
 
-const DropDownViewList = () => {
-  const Categories = [
-    'Categories',
-    'Category 1',
-    'Category 2',
-    'Category 3',
-    'Category 4',
-    'Category 5',
-  ];
-  const SortBy = [
-    'Sort By Most Recent',
-    'Recent Collections',
-    'A-Z',
-    'Z-A',
-    'Price ↓',
-    'Price ↑',
-  ];
-  const AllFilter = [
-    'All Filter',
-    'Price',
-    'Size',
-    'Color',
-    'Reviews',
-    'Brand',
-  ];
+const DropDownViewList = ({
+  selectCategory,
+  selectSortBy,
+  selectAllFilter,
+}) => {
+  const [categories, setCategories] = useState([]);
+  const fetchProducts = () => {
+    fetch(`${apiURL()}/categories/`)
+      .then((res) => res.json())
+      .then((item) => {
+        setCategories(item);
+      });
+  };
+  useEffect(() => {
+    fetchProducts();
+  }, []);
+
+  const Categories = categories.map(
```

Comment:
Name your variables in camelCase, not PascalCase :) So just `categories`.

Reply:
I have updated name of variable.